conode/Makefile

```diff
@@ -1,54 +1,55 @@
 CONTAINER = conode
 IMAGE_NAME = c4dt/$(CONTAINER)
 TAG = dev-$(shell date +%y%m%d)
-WEBAPP_ASSET := ../webapp/src/assets/config.toml
+WEBAPP_ASSET := ../webapp/src/assets/config.local.toml
+BUILD_DIR := build/conodes
+BUILT := $(BUILD_DIR)/built
 
 all: docker
 
-.PHONY: cothority-pull conode newdb docker docker_run docker_wait docker_stop docker_start docker_clean clean
+.PHONY: newdb docker_run docker_wait docker_stop docker_start docker_clean clean
 
-.DEFAULT_GOAL := docker_run
+.DEFAULT_GOAL := $(BUILT)
 
-utils: cothority-pull
-	@echo "Compiling binaries"; \
-	go build -o build/conode; \
-	cd ../cothority && \
-		go build -o ../conode/build/bcadmin ./byzcoin/bcadmin && \
-		go build -o ../conode/build/csadmin ./calypso/csadmin && \
-		go build -o ../conode/build/phapp ./personhood/phapp
+$(BUILT): build/Dockerfile build/run_nodes.sh
+	make newdb
+	make docker
+	touch $(BUILT)
 
-cothority-pull:
-	make -C .. cothority
-
-newdb: utils
-	rm -f build/conodes/*db;
-	@echo "Starting nodes"; \
-	( cd build; COTHORITY_ALLOW_INSECURE_ADMIN=true ./run_nodes.sh -v 2 -t -n 4 -s -d conodes > /dev/null ) & \
-	make docker_wait; \
-	echo "Creating new ByzCoin and printing out important variables"; echo; \
-	rm -f *.cfg; \
-	( ./build/bcadmin -c . create build/conodes/public.toml --interval 500ms; \
-	  ./build/bcadmin latest --bc bc-*; \
-	  ./build/bcadmin key -print key-* ) | egrep "^(ByzCoin|Admin|Private|Creating Spawner)" | tee variables.txt; \
-	echo; echo "Shutting down conodes"; pkill -x conode; rm -f conodes/running
+newdb: docker
+	rm -f build/conodes/*{db,cfg,key}
+	@echo "Starting nodes"
+	docker run -di -v $$(pwd)/build/conodes:/root/conodes --name newdb -p 7770-7777:7770-7777 \
+		$(IMAGE_NAME):dev bash -c \
+		"COTHORITY_ALLOW_INSECURE_ADMIN=true ./run_nodes.sh -n 4 -v 2 -c -d /root/conodes"
+	make docker_wait
+	@echo "Creating new ByzCoin and printing out important variables"
+	docker exec newdb ./bcadmin -c conodes create conodes/public.toml --interval 500ms | tee variables.txt
+	docker exec newdb bash -c './bcadmin key -print $$(ls conodes/key*)' | tee -a variables.txt
+	docker exec newdb bash -c './bcadmin info --bc $$(ls conodes/bc*)' | grep AdminDarc | tee -a variables.txt
+	docker exec newdb chmod -R a+r /root/conodes
```

Comment:
maybe add a comment

Reply:
The comment is in the `@echo` - so it appears also during the `make`...

Reply:
a comment for the dev I mean, to know why a `chmod` is necessary